src/assets/scripts/base/PositionModel.js

```diff
@@ -175,31 +233,17 @@ export default class PositionModel {
     }
 
     /**
-     * The distance in km between two locations
      *
-     * @for PositionModel
-     * @method distanceToPoint
-     * @param lat_a
-     * @param lng_a
-     * @param lat_b
-     * @param lng_b
-     * return {number}
+     *
      */
-    distanceToPoint(lat_a, lng_a, lat_b, lng_b) {
-        const d_lat = degreesToRadians(lat_a - lat_b);
-        const d_lng = degreesToRadians(lng_a - lng_b);
-
-        // TODO: what is actually getting set to `a` here? and what does `a` represent?
-        const a = Math.pow(sin(d_lat / 2), 2) +
-            (
-                cos(degreesToRadians(lat_a)) *
-                cos(degreesToRadians(lat_b)) *
-                Math.pow(sin(d_lng / 2), 2)
-            );
-        // TODO: what is actually getting set to `c` here? and what does `c` represent?
-        const c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
-
-        // TODO: what does this number mean? enumerate the magic nubmer
-        return c * 6371.00;
+    adjustForMagneticNorth() {
+        // Adjust to use magnetic north instead of true north
+        let t = Math.atan2(this.y, this.x);
+        const r = Math.sqrt((this.x * this.x) + (this.y * this.y));
```

Comment:
fantastic!  Math and I are not exactly close, but this diagram really helps.  When it comes to named formulas and such, I'm ok with single letter var names.  Especially if thats how the formula is defined.  I just need to add references to the actual formula.